src/components/renderer/field/form-field-renderer.component.tsx

```diff
@@ -39,9 +39,22 @@ export const FormFieldRenderer = ({ fieldId, valueAdapter, repeatOptions }: Form
   const [warnings, setWarnings] = useState<ValidationResult[]>([]);
   const [historicalValue, setHistoricalValue] = useState<ValueAndDisplay>(null);
   const context = useFormProviderContext();
-  const { hideUnansweredQuestionsInReadonlyForms } = useConfig({
-    externalModuleName: '@openmrs/esm-form-engine-app',
-  });
+
+  // Try to get config from external module, fallback to default if not available
+  let hideUnansweredQuestionsInReadonlyForms = false;
+  try {
+    const config = useConfig({
+      externalModuleName: '@openmrs/esm-form-engine-app',
+    });
+    hideUnansweredQuestionsInReadonlyForms = config?.hideUnansweredQuestionsInReadonlyForms ?? false;
+  } catch (error) {
+    // If external module config is not available, use default value
+    console.warn(
+      'Could not load external module config, using default value for hideUnansweredQuestionsInReadonlyForms:',
```

Comment:
```suggestion
      'Failed to load @openmrs/esm-form engine-app config - using hideUnansweredQuestionsInReadonlyForms=false (empty fields will be visible in readonly mode): '
```

Might be better to explain the practical effect that failing to load the module config has.